cpp/daal/src/data_management/finiteness_checker.h

```diff
@@ -0,0 +1,52 @@
+/** file finiteness_checker.h */
+/*******************************************************************************
+* Copyright 2024 Intel Corporation
```

Comment:
Please align copyrights with the new UXL requirements. 
```suggestion
* Copyright contributors to the oneDAL project
```